cnames_active.js

```diff
@@ -639,6 +639,7 @@ var cnames_active = {
   "css-variable": "jantimon.github.io/css-variable",
   "csv": "adaltas.github.io/node-csv-docs",
   "csv-rex": "willfarrell.github.io/csv-rex",
+  "cua":"chen-rn.github.io/CUA"
```

Comment:
```suggestion
  "cua": "chen-rn.github.io/CUA",
```
Your entry is missing a space, and a comma